OSVectorTileAPI/MapboxGLJS/map.js

```diff
@@ -20,6 +21,8 @@ function setupLayer() {
     message.textContent = 'To view the map, please enter a valid API key.';
     instructions.classList.add("hidden");
 
+    // This sets up the actual VTS layer
+    // Center coordinates are defined in EPSG:3857 lon/lat as we are asking for srs=3857 in the "transformRequest"
```

Comment:
Center coordinates are defined in EPSG:3857 lon/lat **and** we are asking for srs=3857 in the "transformRequest"

- Mapbox is in 3857 anyway, we have to supply it with 3857 data

Reply:
"Mapbox is in 3857 anyway, we have to supply it with 3857 data"
Didn't know that - worth highlighting in the comments